implementations/py/polywrap-http-plugin/tests/test_plugin.py

```diff
@@ -0,0 +1,34 @@
+import json
+from typing import cast
+from polywrap_http_plugin import http_plugin, HttpResponse
+from polywrap_client import PolywrapClient, PolywrapClientConfig
+from polywrap_core import Uri, IUriResolver, InvokerOptions
+from polywrap_uri_resolvers import StaticResolver, RecursiveResolver
+from polywrap_uri_resolvers.uri_resolver_aggregator import UriResolverAggregator
+from polywrap_uri_resolvers.legacy import FsUriResolver, SimpleFileReader
+
+
+async def test_plugin():
```

Comment:
Can we please expand on the testing that's being done here to cover all public methods? Can use the JS tests as a reference: https://github.com/polywrap/http/blob/main/implementations/js/src/__tests__/e2e/e2e.spec.ts